test/mocha/browserstack-capabilities-test.js

```diff
@@ -63,4 +63,38 @@ describe('Browserstack capabilities', function () {
             'browserstack.networkProfile': '4g-lte-lossy'
         });
     });
+
+    it('Should add additional capabilities when both config file and environment variables used', () => {
+        process.env['BROWSERSTACK_BUILD_ID'] = 'build-1';
+        process.env['BROWSERSTACK_PROJECT_NAME'] = 'project-1';
+        process.env['BROWSERSTACK_DISPLAY_RESOLUTION'] = '1024x768';
+        process.env['BROWSERSTACK_TEST_RUN_NAME'] = 'Testcafe test run 1';
+        process.env['BROWSERSTACK_DEBUG'] = 'false'; //env var should take precedence
+        process.env['BROWSERSTACK_CONSOLE'] = 'errors';
+        process.env['BROWSERSTACK_NETWORK_LOGS'] = 'true';
+        process.env['BROWSERSTACK_VIDEO'] = 'true';
+        process.env['BROWSERSTACK_TIMEZONE'] = 'Asia/Taipei';
+        process.env['BROWSERSTACK_GEO_LOCATION'] = 'ZA';
+        process.env['BROWSERSTACK_CUSTOM_NETWORK'] = '"1000", "1000", "100", "1"';
+        process.env['BROWSERSTACK_NETWORK_PROFILE'] = '4g-lte-lossy';
+
+        process.env.BROWSERSTACK_CAPABILITIES_CONFIG_PATH = require.resolve('./data/capabilities-config.json');
+
+        const capabilities = browserStackProvider._getAdditionalCapabilities();
```

Comment:
The current implementation sets an absolute path, but we need to test the functionality of `_getCapabilitiesFromConfig` with a relative path.

When a relative path is passed to `path.resolve()` in `_getCapabilitiesFromConfig`, it returns an absolute path based on the working directory. Therefore, I suggest you save the current working directory, set `__dirname` before retrieving the capabilities, and then restore the original working directory afterwards.


```suggestion
        process.env.BROWSERSTACK_CAPABILITIES_CONFIG_PATH = './data/capabilities-config.json';
        
        const originWorkDir = process.cwd();
        process.chdir(__dirname)
        
        const capabilities = browserStackProvider._getAdditionalCapabilities();
        
        process.chdir(originWorkDir)
```